reference/release-notes/release-notes.md

```diff
@@ -45,18 +46,16 @@ To ensure you receive latest security updates and bug fixes, you should upgrade
 If you are looking for additional support, see [Ubuntu Pro](https://ubuntu.com/support). Canonical can also provide [managed solutions](https://ubuntu.com/managed) for Anbox Cloud.
 
 
-### What's new in 1.23.x?
+### What's new in 1.24.x?
 
-Along with bug fixes and general improvements, Anbox Cloud 1.23.x includes:
+Along with bug fixes and general improvements, Anbox Cloud 1.24.x includes:
 
-* Improved Anbox Cloud Appliance
-* Seamless ADB connections to remote Android instances
-* Dashboard improvements
-    - Merge of sessions and instances pages
-    - Ability to authorise ADB connections
-* Improved documentation delivery
-* Android 14 based vendor images
-* Reworked charms that install snaps from the snap store
+* VHAL improvements
+* Android 14 images
+* Support for generic application streaming
```

Comment:
Does this mean we're going to release a generic image in 1.24.0? if not, do we need to call ou in the release announcement?

Reply:
@morphis Can you confirm if generic application streaming is dropped from 1.24.0?

Reply:
Hmm, we’ve never formally released the generic image, so this isn't about dropping it as of version 1.24.0.

Reply:
@keirthana generic images have never been released and will not any time soon. So lets not mention them at all.